pkg/mobile/app/service.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"github.com/feedhenry/mcp-standalone/pkg/mobile"
+	"github.com/pkg/errors"
+	"github.com/satori/go.uuid"
+)
+
+type Service struct {
+}
+
+func (s *Service) Create(appCrudder mobile.AppCruder, app *mobile.App) error {
+	uid := uuid.NewV4()
+	app.APIKey = uid.String()
+	switch app.ClientType {
+	case "android":
```

Comment:
probably could extract these to constants as they are very likely to be used again

Reply:
@maleck13 Would the constants stay in this package or move to types.go?